### PR TITLE
kubetest: support reusing prebuilt kops version

### PR DESCRIPTION
**Pull request by justinsb**

### CODE REVIEW

**Comment by justinsb** [comment]
Builds on #5117 (first commit).  Second commit is the relevant one here.

This incorporates some of the functionality currently in kops-e2e-runner.sh into kubetest instead.  We can pass a `--kops-version`, which is a URL containing a `--kops-base-url`, and we'll download kops from there.  We can also specify the flags directly and bypass the "chain"

File: scenarios/kubernetes_e2e.py [hunk 1]
```diff
@@ -689,7 +717,12 @@ def create_parser():
         default=[],
         help='Send unrecognized args directly to kubetest')
 
-    # aws
+    # kops & aws
+    # At the moment, the logic is that `--aws` => kops & aws,
+    # and `--kops` => kops & gce.
+    # TODO(justinsb): get this using a `--provider` flag, just like kubetest
+    parser.add_argument(
+        '--kops', action='store_true', help='E2E job runs using kops')
```
Comment (krzyzacy) [inline scenarios/kubernetes_e2e.py]: `--kops-gce`? `--kops` is somewhat confusing, probably just change it to `--provider`, and pass through to kubetest

File: scenarios/kubernetes_e2e.py [hunk 2]
```diff
@@ -724,9 +757,13 @@ def create_parser():
         '--kops-state', default='s3://k8s-kops-jenkins/',
         help='Name of the aws state storage')
     parser.add_argument(
+        '--kops-state-gce', default='gs://k8s-kops-gce-jenkins/',
+        help='Name of the kops state storage for GCE')
+    parser.add_argument(
         '--kops-zones', help='Comma-separated list of zones else random choice')
     parser.add_argument(
         '--kops-build', action='store_true', help='If we need to build kops locally')
+    parser.add_argument('--kops-version', help='URL containing kops version to test')
```
Comment (krzyzacy) [inline scenarios/kubernetes_e2e.py]: seems scenario did not use this var - you can drop the flag and it will pass through to kubetest

**Comment by justinsb** [comment]
(I think the problem is that aws relies on HOME being set, if aws_ssh, aws_pub & aws_cred aren't set, and HOME isn't set for bazel tests, and this was the scenario I was testing in the expanded arg-passthrough test)

**Comment by justinsb** [comment]
And it's finally green again!

**Comment by krzyzacy** [comment]
/lgtm
I'll bump the image for the kops-gce job

**Pull request merged**